packages/wrangler/src/init.ts

```diff
@@ -88,6 +88,7 @@ export const init = createCommand({
 			const accountId = await requireAuth({});
 			try {
 				await fetchResult<ServiceMetadataRes>(
+					undefined,
```

Comment:
`wrangler init` should not be run from inside a project - sort of defeats its purpose. So we explicitly do not try to load any Wrangler config and rely only on the `CLOUDFLARE_COMPLIANCE_REGION` env var.

Reply:
This is now set to `COMPLIANCE_REGION_UNKNOWN` to make it more self-documenting.